src/cascadia/TerminalSettingsEditor/ColorSchemes.xaml

```diff
@@ -36,176 +36,21 @@
         </ResourceDictionary>
     </Page.Resources>
 
-    <ScrollViewer ViewChanging="ViewChanging">
-        <StackPanel Style="{StaticResource SettingsStackStyle}">
-            <TextBlock x:Uid="ColorSchemesDisclaimer"
+    <Grid RowSpacing="8"

```

Comment:
note to reviewers: hit that <kbd>.</kbd>. The GH diff is egregious compared to the vscode one.